packages/discord.js/typings/index.d.ts

```diff
@@ -4206,7 +4210,7 @@ interface GuildAuditLogsTypes {
   ChannelDelete: ['Channel', 'Delete'];
   ChannelOverwriteCreate: ['Channel', 'Create'];
   ChannelOverwriteUpdate: ['Channel', 'Update'];
-  ChannelOverwriteDelete: ['CHANNEL', 'Delete'];
+  ChannelOverwriteDelete: ['Channel', 'Delete'];
```

Comment:
I couldn't find any reason as to why this was uppercase. If there is a reason then LMK